### ISSUE TITLE
azure-identity

### ISSUE THREAD

**Issue by DeepakPradeepKumar** [Azure/azure-sdk-for-python#32396]
Why is it so difficult to import azure modules and work with them in Python (VSCode) in the first place ? all the packages start throwing one or the other error.
azure-identity -> modulenotfounderror -> irrespective of uninstalling and reinstalling Python and dependent packages, still unable to resolve the issue.
azure-cosmos -> conda envrionment doesn't even find this in the right channel, only for this we have to have a separate channel. 
azure-storage and keyvault -> these have separate issues.

currently, i ma having this issue- from azure.identity import DefaultAzureCredential
ModuleNotFoundError: No module named 'azure.identity'

I have both Pip & Conda, and tried changing the envrionments and interpreters in VSCode. also installed Azure authenticator and signed in, tried all the possible steps mentioned in the documentation, still having trouble getting this resolved.

please help.

**Comment by annatisch** [comment]
Thanks for the report @DeepakPradeepKumar - sorry you're having trouble with the packages in intellisense in VSCode.

Are you able to confirm whether any other packages you've installed (i.e. not included by default in the environment) are displaying accurately in VSCode? Is `azure-identity` the only Azure package giving the "ModuleNotFound" error? In other words, `storage` and `keyvault` are found in the environment, but just not `identity`?

Hopefully we can clarify the nature of the issue so we can help investigate ASAP. :)

**Comment by DeepakPradeepKumar** [comment]
@joshfree : Thank you for the response, appreciate it. I did have a tuff time getting through this.. however, I had to uninstall the python, Anaconda environments and then restart only with installing Python, update all the modules to be installed in requirements.txt and then run the file to make sure that it installs everything from the scratch. Finally got it working.
However, it would be great to make the azure related module installation & accessibility easy and also see why VSCode was not able to recognize the installed modules, even though I installed all azure related modules and validated it again and again before importing it in Python.
